.github/workflows/build.yml

```diff
@@ -63,189 +70,198 @@ jobs:
+          tar -zcvf "../linux_amd64_${{ matrix.config.compiler }}.tar.xz" *
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: linux_amd64_${{ matrix.config.compiler }}.tar.xz
+          path: linux_amd64_${{ matrix.config.compiler }}.tar.xz
+          retention-days: 7
 
   windows:
     name: ${{ matrix.config.name }}
     runs-on: windows-latest
-    env:
-      CONAN_USER_HOME: "${{ github.workspace }}/conan/"
-      CONAN_USER_HOME_SHORT: "${{ github.workspace }}/conan/s"
     strategy:
       fail-fast: false
       matrix:
         config:
           - {
             name: "Windows MSVC",
-            cc: "cl", cxx: "cl",
             compiler: "msvc",
-            build_type: "Release",
-            cmake_build_options: "--config Release",
+            cc: "cl", cxx: "cl",
             cmake_configure_options: '-G "Visual Studio 17 2022" -A x64',
-            conan_profile_update: ''
           }
           - {
             name: "Windows Clang",
-            cc: "clang-cl", cxx: "clang-cl",
             compiler: "clang",
-            build_type: "Release",
-            cmake_build_options: "--config Release",
+            cc: "clang-cl", cxx: "clang-cl",
             cmake_configure_options: '-G "Visual Studio 17 2022" -A x64 -T "LLVM_v143" -DCMAKE_CXX_COMPILER="clang-cl.exe" -DCMAKE_C_COMPILER="clang-cl.exe" -DCMAKE_LINKER="lld.exe"',
-            conan_profile_update: 'conan profile update settings.compiler.libcxx=libstdc++11 default',
           }
 
     steps:
       - name: Update environment
         shell: pwsh
         run: |
           pip3 install wheel setuptools
-          pip3 install conan
 
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Cache Conan Dependencies
-        uses: actions/cache@v3
-        with:
-          path: ${{ env.CONAN_USER_HOME }}
-          key: conan-${{ runner.os }}-${{ matrix.config.name }}-${{ matrix.config.build_type }}-${{ hashFiles('conanfile.py') }}
-
       - name: Configure LLVM
         if: matrix.config.compiler == 'clang'
         shell: pwsh
         run: |
           choco upgrade --no-progress llvm
-          curl -fsSL -o "LLVM_VS2017.zip" "https://github.com/zufuliu/llvm-utils/releases/download/v22.09/LLVM_VS2017.zip"
+          curl -fsSL -o "LLVM_VS2017.zip" "https://github.com/zufuliu/llvm-utils/releases/download/v23.03/LLVM_VS2017.zip"
           7z x -y "LLVM_VS2017.zip" >NUL
           LLVM_VS2017\install.bat
-          conan config init
-          (Get-Content "$(conan config home)\settings.yml") -replace 'v143', 'v143, LLVM_v143' | Out-File -encoding ASCII "$(conan config home)\settings.yml"
 
-      - name: Configure CMake
+      - name: Install Vulkan SDK
+        shell: pwsh
+        run: |
+          curl -LS -o vulkansdk.exe `
+            https://sdk.lunarg.com/sdk/download/${{ env.INEXOR_VULKAN_VERSION }}/windows/VulkanSDK-${{ env.INEXOR_VULKAN_VERSION }}-Installer.exe?Human=true
```

Comment:
Human=true ?

Reply:
I think we can leave that out. It probably came from just copying the URL of the Vulkan SDK.

Reply:
I left it out and it's still working.